src/ontology/edits/oeo-model.omn

```diff
@@ -1047,13 +1047,42 @@ Class: OEO_00010255
         <http://purl.obolibrary.org/obo/IAO_0000118> "NDC",
         <http://purl.obolibrary.org/obo/IAO_0000233> "issue: https://github.com/OpenEnergyPlatform/ontology/issues/947
 pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1151",
-        rdfs:label "nationally determined contribution"@de
+        rdfs:label "nationally determined contribution"
     
     SubClassOf: 
         OEO_00020093,
         <http://purl.obolibrary.org/obo/IAO_0000136> some OEO_00000199
     
     
+Class: OEO_00010260
+
+    Annotations: 
+        <http://purl.obolibrary.org/obo/IAO_0000115> "Parameterisation is the process of translating assumptions into something the model can use.",
+        <http://purl.obolibrary.org/obo/IAO_0000118> "parametrise",
+        <http://purl.obolibrary.org/obo/IAO_0000233> "issue: https://github.com/OpenEnergyPlatform/ontology/issues/1040
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1216",
+        rdfs:label "parameterisation"
+    
+    SubClassOf: 
+        <http://purl.obolibrary.org/obo/BFO_0000015>,
+        <http://purl.obolibrary.org/obo/RO_0002233> some OEO_00000063
+    
+    
+Class: OEO_00010261
+
+    Annotations: 
+        <http://purl.obolibrary.org/obo/IAO_0000115> "Model calibration is a process of varying variables so that the model reproduces the known data (e.g. historical) as far as possible and thus a part of validation.",
```

Comment:
Does one vary variables in the model until historical data is reproduced? I thought one would vary the parameters (exogenous data) until the model reproduces historic values. Then the model is calibrated towards the past given specific parameter settings that may then be varied when looking into the future..?

Reply:
I am fine with _varying exogenous data_ instead of _varying variables_.